assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -74,20 +76,27 @@ const IconWrapper = styled.div`
 // Since the svg-s are 32px and have extra room around, this needs to be removed
 const Collapse = styled.button`
   border: none;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
   background: transparent;
-  color: inherit;
+  color: ${({ theme, isCollapsed }) =>
+    isCollapsed ? theme.colors.fg.secondary : theme.colors.fg.primary};
   height: 32px;
   display: flex; /* removes implicit line-height padding from child element */
   padding: 0 4px 0 0;
   cursor: pointer;
+  margin-left: -12px;
+  transition: ${BUTTON_TRANSITION_TIMING};
+
+  &:hover,
+  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR}, &[data-focus-visible-added] {
+    color: ${({ theme }) => theme.colors.fg.primary};
+  }
+  ${themeHelpers.focusableOutlineCSS};
```

Comment:
This looks so good. The focus styling is layered box shadows, but the inner box shadow defaults to the primary background color. If we want the inner black border to match the panel background we could do:
```
${({ theme }) => 
    themeHelpers.focusableOutlineCSS(
        theme.colors.border.focus, 
        theme.colors.bg.secondary
    )};
```

We haven't figured out a way to have the focus styles inherit the color for the inner box shadow yet 😭. 

![image](https://user-images.githubusercontent.com/22185279/111801370-80a79080-8892-11eb-8cc2-e74f71e345f9.png)

Reply:
Hm, it would ideally be transparent + only the blue line so it would work on top of any background. Perhaps that's something that could have its own issue for adjusting this from the design-system directly.

Reply:
if we need it to be 'transparent' there might be a use case for a different focusableOutline shared style that isn't centered on traditional buttons/focus content. Since this is getting applied directly on the panel title it seems like a good use case for say, not having the break that would need the "transparent" since it doesn't do anything for the focus style to have that break. 

tl;dr is that with the focus style as it is there _must_ be a color given for that first value, this has been the topic of some lengthy discussions with sam and the color between content and focus-blue border is a known trade off.  Just sayin', instead of reworking that we might as well have focus variant for situations like this that don't benefit at all from the extra 2px space. 

thoughts?

Reply:
Would be so great to have it be transparent, however if it's transparent then it'll show the blue box shadow that's underneath it 😅 

We're really using the two box shadows to show an illusion that there's a border 2px away from the content but they both begin at the content edge.

Reply:
Thanks for the context, makes sense. Yep, if that's something that's not an easy change then we could potentially have a variant or perhaps just accept `backgroundColor`.

I'll just change this to the correct colour for now 👍 

Could we change the default colour for `darkTheme` to `theme.colors.bg.secondary`? It looks like the input is mostly used on the panel anyway. Ah -- I suppose that would change it for the `light` theme as well though. Hm, what if we create a new color code for the themes (e.g. something like `focusHelperLine`, surely there's a better name 😛 ) which for the editor would be `theme.colors.bg.secondary` and for the DB `theme.colors.bg.primary` and use this inside `focusableOutlineCSS` instead? Thoughts?

Reply:
i think we're gonna have issues with any of these! I vote we table it until we can see all of the use cases for it and then make a choice on how to clean this up separately with broader context. I still think we might benefit from a version that just cuts out the "transparent" (when i say transparent i just mean from a design perspective) for things like this because the space isn't even a think we need on these cases.

Reply:
Alright, sgtm!